charmcraft.yaml

```diff
@@ -3,11 +3,7 @@ parts:
   charm:
     source: src
     plugin: reactive
-    # using charm/3.x/stable will cause the charm unable to install on 18.04
-    # with the following error: "pip requires Python '>=3.7' but the running
-    # Python is 3.6.9". This is because charm/3.x/stable start using system
-    # installation of python instead of a python snap.
```

Comment:
If you're updating this, could you confirm this comment no longer applies, and that the charm successfully installs on ubuntu 18.04? Or if we're dropping 18.04 support, we should figure out what other changes are required to stop charmhub from advertising 18.04 support https://charmhub.io/openstack-service-checks

Also I don't quite understand the commit message - regardless of CI or local build, this is something that happens internal to the build with charmcraft and should be consistent?

Reply:
@gabrielcocenza was this change still required for something?